### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by xcixor**
#### Purpose of this PR
The purpose of this PR is to write failing tests that define the desired features of the application

#### Tasks to be completed
* Write tests for api endpoints 
* Write tests for Ride model
* Write tests for User mode
* Write tests for Controller 

#### Manual Testing
* Clone this repo
* Check out to the tests branch 
* Create a virtual env and install (pytests)[https://docs.pytest.org/en/3.2.5/getting-started.html]
* Run pytest tests/ to run all the tests in the tests directory

#### Relevant Pivotal Tracker stories
- [Write tests for application #158575425](https://www.pivotaltracker.com/n/projects/2179559)

### CODE REVIEW

**Review by JoshuaOndieki — CHANGES_REQUESTED** [review]
Here's how you can better name your tests :arrow_right: http://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html

TL;DR
- Test name should express a specific requirement.
- Test name should include the expected input or state and the expected result for that input or state.
- Test name should be presented as a statement or fact of life that expresses workflows and outputs.
- Test Name should only begin with Test if it is required by the testing framework or if it eases development and maintenance of the unit tests in some way.
- Test name should include the name of tested method or class.
- Variable names should express the expected input and state.

**Comment by xcixor** [comment]
Thanks for that insight, Let me review that @JoshuaOndieki

**Review by JoshuaOndieki — APPROVED** [review]
:+1: Great effort on the test naming.
